project/plugins.sbt

```diff
@@ -10,7 +10,7 @@ resolvers += Resolver.url("heroku-sbt-plugin-releases",
 // Sbt plugins
 addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.4.2")
 
-addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.5")
+addSbtPlugin("org.scala-js" % "sbt-scalajs" % "0.6.8")
```

Comment:
We should upgrade to Scala.js `0.6.9` - it's the latest one, released a week ago or something.